gitlab-cpu-pipeline.yml

```diff
@@ -0,0 +1,121 @@
+stages:
+    - pre_build
+    - build 
+    - test
+    - performance_eval
+        
+ci_builder_pre:
+    stage: pre_build
+    tags:
+        - ci_testing_cc
+    variables:
+        GIT_STRATEGY: clone
+    before_script:
+        - git branch -vva
+        - echo $commit_author_name
+    script:
+        - echo "checking availability of ci_testing_cc"
+        - git --version
+        - git submodule set-url submodules/xdmfwriter https://github.com/TUM-I5/XdmfWriter.git
+        - git submodule set-url submodules/cxxtest https://github.com/CxxTest/cxxtest.git
+        - git submodule set-url submodules/utils https://github.com/TUM-I5/utils.git
+        - git submodule set-url submodules/async https://github.com/TUM-I5/ASYNC.git
+        - git submodule set-url submodules/scons-tools https://github.com/TUM-I5/scons-tools.git
+        - git submodule set-url submodules/PUML https://github.com/TUM-I5/PUML2.git
+        - git submodule set-url submodules/yateto https://github.com/SeisSol/yateto.git
+        - git submodule set-url submodules/eigen3 https://github.com/eigenteam/eigen-git-mirror
+        - git submodule set-url submodules/Device https://github.com/SeisSol/Device.git
+        - git submodule --quiet update --init --recursive 
+        - wget -q https://syncandshare.lrz.de/dl/fiJNAokgbe2vNU66Ru17DAjT/netcdf-4.6.1.tar.gz
```

Comment:
By the way, is this link actually stable? Most syncandshare links only work for a few months now

Reply:
I don't know who added this and why it was done in this way. I guess @VMPW did it. It is better to ask her

Reply:
Let's just keep it like this, it's an easy fix in the worst case